src/main/java/eu/europeana/metis/sandbox/controller/PatternAnalysisController.java

```diff
@@ -178,8 +180,22 @@ private DatasetProblemPatternAnalysisView(DatasetProblemPatternAnalysis<T> datas
       this.executionTimestamp = datasetProblemPatternAnalysis.getExecutionTimestamp() == null ? null :
           datasetProblemPatternAnalysis.getExecutionTimestamp().toString();
       this.problemPatternList = datasetProblemPatternAnalysis.getProblemPatternList();
+      orderRecordListsInProblemPatternList();
       problemPatternList.sort(Comparator.comparing(elem -> elem.getProblemPatternDescription().getProblemPatternId()));
     }
+
+    private void orderRecordListsInProblemPatternList(){
```

Comment:
You can cleanup this code if the "shorhand" using streams solves the issue.

Reply:
As I mentioned above, the suggested block of code does not do the same thing and this private method.
Would like to hear suggestions if you have any

Reply:
see the suggestion.